nbextensions/config/main.js

```diff
@@ -126,10 +115,10 @@ define([
      */
     function set_config_active (extension, state) {
         state = state === undefined ? true : state;
-        console.log('nbext', state ? ' enable:' : 'disable:' , extension.Name );
+        console.log('nbext', state ? ' enable:' : 'disable:', extension.Name);
```

Comment:
I think just something like `Notebook extension <name> enabled/disabled` would be enough.